libnmstate/nm/ipv4.py

```diff
@@ -119,3 +122,19 @@ def get_ip_profile(active_connection):
     if remote_conn:
         return remote_conn.get_setting_ip4_config()
     return None
+
+
+def get_route_info():
+    client = nmclient.client()
+    skip_networks = [iplib.IPV4_LINK_LOCAL_NETWORK]
+    func = {
```

Comment:
This looks like an attempt to push everything into one handler instead of splitting things up into multiple handlers which are called from here to process the needed.
Sometimes, especially when there is a complex logic, this makes sense, but here I'm not sure it is needed in this case. The first smell will be that this func does too many things.
Even if it is needed, using a dict to collect the funcs is not that nice. It will be nicer to use a direct argument variable.

Reply:
Thanks for the suggest, I will try find a better way.